examples/nlp/lasertagger/official_lasertagger/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Ok, this file should be deleted FOR SURE.

Please note we have Apache LICENSE file already in main NeMo folder...

Reply:
will remove it